src/main/cpp/include/utils/vid_mapper_pb.h

```diff
@@ -25,12 +25,32 @@
 #include "vid_mapper.h"
 #include "genomicsdb_vid_mapping.pb.h"
 #include "genomicsdb_callsets_mapping.pb.h"
+#include <google/protobuf/stubs/common.h>
 
 enum {
   GENOMICSDB_VID_MAPPER_SUCCESS = 0x0,
   GENOMICSDB_VID_MAPPER_FAILURE = 0x1
 };
 
+/*
+ * Constructor is called for the global variable when the library is loaded
+ * Destructor is called when the library is unloaded;
+ */
+class GenomicsDBProtoBufInitAndCleanup
+{
+  public:
+    GenomicsDBProtoBufInitAndCleanup() { }
+    ~GenomicsDBProtoBufInitAndCleanup()
+    {
+      GenomicsDBProtoBufInitAndCleanup::shutdown_protobuf_library();
+    }
+    static void shutdown_protobuf_library()
+    {
+      google::protobuf::ShutdownProtobufLibrary();
```

Comment:
It's not really a problem - valgrind gives a bunch of messages if this is not called. I'm afraid that if I don't get rid of these messages, then I will miss a real memory leak/error message sometime later.